src/main/kotlin/org/cascadebot/cascadebot/data/objects/GuildCommandInfo.kt

```diff
@@ -0,0 +1,28 @@
+package org.cascadebot.cascadebot.data.objects
+
+import org.cascadebot.cascadebot.commandmeta.ICommandMain
+import org.cascadebot.cascadebot.data.language.Locale
+
+class GuildCommandInfo(var enabled: Boolean,
+                       command: String,
+                       defaultCommand: String,
+                       var aliases: MutableSet<String>) {
```

Comment:
This class needs to be updated to the new decision to not allow changing of commands

Reply:
Yeah I half did that, I will finish that 👍

Reply:
@weeryan17 Done! :)